File: app/src/main/java/com/example/homework_m2_6/MainActivity.java
package com.example.homework_m2_6;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {


    private EditText editTextText;
    private EditText editTextText2;
    private EditText editTextText3;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        editTextText = findViewById(R.id.editTextText);
        editTextText2 = findViewById(R.id.editTextText2);
        editTextText3 = findViewById(R.id.editTextText3);
    }

    public void buttonSend(View view) {
        Intent intent = new Intent(Intent.ACTION_SENDTO);
        intent.setData(Uri.parse("mailto:"));
        intent.putExtra(Intent.EXTRA_EMAIL, new String[]{editTextText.getText().toString()});
        intent.putExtra(Intent.EXTRA_SUBJECT, editTextText2.getText().toString());
        intent.putExtra(Intent.EXTRA_TEXT, editTextText3.getText().toString());
        startActivity(intent);
    }

    public void onNextButtonClick(View view) {
        finish();
    }
}

File: app/src/main/java/com/example/homework_m2_6/MainActivity.kt
package com.example.homework_m2_6

import android.graphics.Color
import android.os.Bundle
import android.view.View
import android.text.Editable
import android.text.TextWatcher
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
class MainActivity : AppCompatActivity() {
    private lateinit var editText: EditText
    private lateinit var button: Button
    private lateinit var editTextEmail: EditText
    private lateinit var editTextPassword: EditText
    private lateinit var buttonLogin: Button
    private lateinit var textViewWelcome: TextView
    private lateinit var textView2: TextView
    private lateinit var textView3: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        editTextEmail = findViewById(R.id.editTextText)
        editTextPassword = findViewById(R.id.editTextText2)
        buttonLogin = findViewById(R.id.button)
        textViewWelcome = findViewById(R.id.textView)
        textView2 = findViewById(R.id.textView2)
        textView3 = findViewById(R.id.textView3)


        buttonLogin.setOnClickListener {
            validateCredentials()
        }



        editText = findViewById(R.id.editTextText)
        button = findViewById(R.id.button)

        editText.addTextChangedListener(object : TextWatcher {
            override fun beforeTextChanged(charSequence: CharSequence?, start: Int, before: Int, count: Int) {
            }

            override fun onTextChanged(charSequence: CharSequence?, start: Int, before: Int, count: Int) {
                if (!charSequence.isNullOrBlank()) {
                    button.setBackgroundColor(resources.getColor(R.color.orange))
                } else {
                    button.setBackgroundColor(resources.getColor(R.color.gray))
                }
            }

            override fun afterTextChanged(editable: Editable?) {
            }
        })






    }
    fun validateCredentials() {
        val email = editTextEmail.text.toString()
        val password = editTextPassword.text.toString()

        if (email == "admin" && password == "admin") {
            showToast("Вы успешно зарегистрировались")
            editTextEmail.visibility = View.GONE
            editTextPassword.visibility = View.GONE
            buttonLogin.visibility = View.GONE
            textView2.visibility = View.GONE
            textView3.visibility = View.GONE

        } else {
            showToast("Неправильный логин или пароль")
        }
    }

    private fun showToast(message: String) {
        Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
    }
}